PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/GraphTest.java

```diff
@@ -0,0 +1,34 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public abstract class GraphTest {
+  
+  public GraphInterface gra;
```

Comment:
Don't needlessly use very short illogical variable names (i.e. use "graph" in this case).